docs/process/testing.md

```diff
@@ -1,366 +1,491 @@
 ### Download Desktop
-  - [ ] User can download latest (Mac & Windows) Desktop from https://desktop.github.com/    
-    - [ ] Mac: https://central.github.com/deployments/desktop/desktop/latest/darwin
-      - [ ] Homebrew package manager: $ brew cask install github-desktop
-    - [ ] Windows: https://central.github.com/deployments/desktop/desktop/latest/win32
-      - [ ] Chocolatey package manager: c:\> choco install github-desktop
-      - [ ] 64-bit and up
-    - [ ] Data is retained if you download and open a fresh copy
-  - [ ] Release notes page is up-to-date
-  - [ ] Help page is accessible
-  - [ ] 'Please update' notifcation shown in Classic apps
+
+* [ ] User can download latest (Mac & Windows) Desktop from
+      https://desktop.github.com/
+  * [ ] Mac:
+        https://central.github.com/deployments/desktop/desktop/latest/darwin
+    * [ ] Homebrew package manager: $ brew cask install github-desktop
+  * [ ] Windows:
+        https://central.github.com/deployments/desktop/desktop/latest/win32
+    * [ ] Chocolatey package manager: c:\> choco install github-desktop
+    * [ ] 64-bit and up
+  * [ ] Data is retained if you download and open a fresh copy
+* [ ] Release notes page is up-to-date
+* [ ] Help page is accessible
+* [ ] 'Please update' notifcation shown in Classic apps
```

Comment:
@tierninho Are these changes compatible with Quality, or should I ignore this file?